src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -86,7 +86,15 @@
     static {
         Properties prop = new Properties();
         try {
-            prop.load(new FileInputStream("src/test/resources/test.properties"));
+            String propertiesFilename;
+            if (isTravis()) {
+                propertiesFilename = "test.travis.properties";
+            } else if (isAppveyor()) {
+                propertiesFilename = "test.appveyor.properties";
+            } else {
+                propertiesFilename = "test.properties";
+            }
+            prop.load(new FileInputStream("src/test/resources/" + propertiesFilename));
```

Comment:
E.g. are we really going to hardcode the path to the chromedriver binary in the AppVeyor environment inside our codebase? We'd have to change the code whenever their environment changes.